Cargo.toml

```diff
@@ -0,0 +1,2 @@
+[dependencies]
```

Comment:
Should we use this instead?
[target.'cfg(windows)'.dependencies]

Reply:
Will fix that and re-PR shortly.